psk_fitness/psk_fitness/Controllers/ExerciseController.cs

```diff
@@ -37,6 +36,28 @@ public async Task<ActionResult<ExerciseDisplayDTO>> GetExerciseAsync(int id)
             return Ok(exercise);
         }
 
+        [HttpGet("for-workout/{userEmail}")]
+        public async Task<ActionResult<List<ExerciseForWorkoutDTO>>> GetExerciseForWorkout([FromRoute] string userEmail)
```

Comment:
The name implies that it gets one Exercise, not a list. Again, a user can have many workouts, so the parameters should depend on workouts themselves.

Reply:
Replied to original question about it.